api/app.js

```diff
@@ -76,10 +80,7 @@ let verifySession = (req, res, next) => {
         user.sessions.forEach((session) => {
             if (session.token === refreshToken) {
                 // check if the session has expired
-                if (User.hasRefreshTokenExpired(session.expiresAt) === false) {
-                    // refresh token has not expired
-                    isSessionValid = true;
-                }
+                isSessionValid = Boolean(User.hasRefreshTokenExpired(session.expiresAt))
```

Comment:
The if wasn't neccesary, since we can just cast the return to a Boolean based on if it returns a truthy or falsy value